crates/ra_assists/src/handlers/expand_glob_import.rs

```diff
@@ -133,28 +147,38 @@ fn replace_ast(
     path: ast::Path,
     used_names: Vec<Name>,
 ) {
-    let new_use_tree_list = ast::make::use_tree_list(used_names.iter().map(|n| {
-        ast::make::use_tree(ast::make::path_from_text(&n.to_string()), None, None, false)
-    }));
+    let replacement: Either<ast::UseTree, ast::UseTreeList> = if used_names.len() == 1 {
```

Comment:
This can be rewritten as:
```suggestion
    let replacement: Either<ast::UseTree, ast::UseTreeList> = match used_names.first() {
```

...to avoid the `.first().unwrap()` below

Reply:
matching on `.first()` wouldn't do because i need a different case (use `UseTree` for one element, use `UseTreeList` for more than one elements) if vec has more than one element

but i can pattern match on slice elements
```rust
let replacement: Either<ast::UseTree, ast::UseTreeList> = match used_names.as_slice() {
    [name] => Either::Left(ast::make::use_tree(
        ast::make::path_from_text(&format!("{}::{}", path, name)),
        None,
        None,
        false,
    )),
    names => Either::Right(ast::make::use_tree_list(names.iter().map(|n| {
        ast::make::use_tree(ast::make::path_from_text(&n.to_string()), None, None, false)
    }))),
};
```
is this good?

Reply:
That looks nice, yeah